test/__snapshots__/DatePicker.test.tsx.snap

```diff
@@ -310,12 +314,6 @@ Object {
             </legend>
           </fieldset>
         </div>
-        <p
-          class="MuiFormHelperText-root Mui-error MuiFormHelperText-sizeMedium MuiFormHelperText-contained Mui-required css-1wc848c-MuiFormHelperText-root"
-          id="mui-9-helper-text"
-        >
-          date must be a \`date\` type, but the final value was: \`Invalid Date\`.
-        </p>
```

Comment:
regression?

Reply:
what is a regression? (I'm totally new to jest!)

I mentioned in one comment that basically all the DatePicker... component tests are likely broken with this change and need some further investigation into why they aren't working as expected.

I wonder if awaiting the expect would help with these

Reply:
Sorry for the late reply... yea, it could just be a mistake on my part to have included it, but it *smells* to me like a regression. Would have to find the corresponding test and check.

Reply:
resolved